src/graphene-point.c

```diff
@@ -295,7 +295,11 @@ graphene_point_to_vec2 (const graphene_point_t *p,
   v->value = graphene_simd4f_init (p->x, p->y, 0.f, 0.f);
 }
 
+#ifdef _MSC_VER
+static const graphene_point_t _graphene_point_zero = {0.f, 0.f};
+#else
 static const graphene_point_t _graphene_point_zero = GRAPHENE_POINT_INIT_ZERO;
```

Comment:
This is kind of weird, considering that the macro expands to the same thing.

I assume it's the `(type)` cast annotation that breaks it and makes the compiler think this is not a constant assignment.